lib/ansible/modules/identity/ipa/ipa_dnsrecord.py

```diff
@@ -242,7 +247,7 @@ def ensure(module, client):
 
 
 def main():
-    record_types = ['A', 'AAAA', 'A6', 'CNAME', 'DNAME', 'PTR', 'TXT']
+    record_types = ['A', 'AAAA', 'A6', 'CNAME', 'DNAME', 'PTR', 'TXT', 'SRV']
```

Comment:
Would you be interested in adding support for 'MX' record as well ? You can find feature request here - https://github.com/ansible/ansible/issues/41842

Reply:
Sure, I can take a look.